sig-and-wg/sig-core/proposals/introducing_knative_brokers.md

```diff
@@ -0,0 +1,90 @@
+# Integration of the Knative-based function controller
+
+## Introduction
+
+As of 0.5, Knative Eventing defines [`Broker` and `Trigger`](https://knative.dev/docs/eventing/broker-trigger/) objects as higher level abstractions that allows brokering events based on some event filtering criterea.
+
+This is a proposal of how to introduce these objects in Kyma as an alternative to using the existing Kyma Event Bus. It describes how to build namespace scoped eventing mesh with fault domain isotation and access control. The proposal describes two possible solutions, the migration path and tries to highlight and address the challenges and open questions of using them.
+
+## Features
+
+### Current Features
+
+- As a developer, I want to publish events to Kyma from outside the cluster.
+- As a developer, I want to publish events to Kyma from inside the cluster.
+- As a developer, I want to subscribe to events in Kyma from a specific source with a specific event-type.
+- As an admin, I want to enable or disable events in a specific namespace.
+
+### New features
+
+- As an admin, I want to configure the default cluster wide Knative `Channel` kind to be used. 
+- As an admin, I want to configure a Knative `Channel` kind to be used by default for events per namespace that overrides the cluster default configuration.
+
+## Proposed solution
+
+There are two proposed solutions. Solution 1 is an instant transition and deprecation proposal where the new `Broker` based eventing will replace the existing Event Bus. The Kyma `Susbscription` and `EventActivation` CRDs will be removed and Event Bus users will need to migrate to the new knative CRDs.
```

Comment:
- `new Broker/Trigger` based
- `EventActivation` CRDs will not be removed now until we solve the Event registry workflow.